BUILD

```diff
@@ -67,6 +67,8 @@ cc_library(
     hdrs = [
         ":gemmlowp_public_headers",
     ],
+    includes = ["."],
+    linkopts = ["-pthread"],
```

Comment:
-pthread and not -lpthread?  Is there a special-casing of pthread by the linker here?